### DISCUSSION TITLE
Floating window becomes tiled when toggling fullscreen on and off

### DISCUSSION THREAD

**Discussion by mcnuggetsx20** [qtile/qtile#4410] [Ideas]
This happens for example when I watch youtube - I have my browser in  ```floating``` layout and I make the video fullscreen, which apparently resizes the browser window to fill the whole screen. When I go back from fullscreen it remains resized. Now when I do the same thing but instead I'm in a group with some tiling layout, and I toggle the browser state to ```floating```, then turning off fullscreen tiles it back.
I honestly don't know if that's intended so I decided not to post it in the Issues tab.

**Comment by ramnes** [comment]
Yeah I think that's the same bug as these issues? #2179 #2163

**Reply by mcnuggetsx20** [reply]
Yes, it does look like those two. I'm a little worried by their dates though. They're quite old and no one seems keen on resolving them. I guess this one needs to be marked as a duplicate.

**Reply by ramnes** [reply]
We don't have a "duplicate" label, we usually just close the discussions. That being said, it's not an exact duplicate; the issues I mentioned are solely about mpv when your problem occurs with a browser. (Which one?) Can you open a new issue with more details and cross-reference these other issues inside?

As to the date of the issues, well, yeah, but I think the fix here shouldn't be too hard, so give it a try! PR (and new contributors) are always welcome. :)
